### ISSUE TITLE
ANDROID_LINT_JARS environment doesn't work

### ISSUE THREAD

**Issue by gosp** [googlesamples/android-custom-lint-rules#50]
download the latest code, and remove `implementation project(':library')` in `app/build.gradle`; and copy `checks.jar` to /tmp

```
osboxes# export ANDROID_LINT_JARS=/tmp/checks.jar 
osboxes# ./gradlew clean :app:build              

> Task :app:lintDebug

Scanning app: ...
No issues found.

> Task :app:copyDebugAndroidLintReports
Copying lint SARIF report to /home/osboxes/Documents/android-custom-lint-rules/lint-results.sarif

BUILD SUCCESSFUL in 3s
76 actionable tasks: 74 executed, 2 up-to-date
```

**Comment by gosp** [comment]
It works after I downgrade `gradlePluginVersion` from `7.0.0-alpha10` to `4.2.1`; `lintVersion` from `30.0.0-alpha10` to `27.2.1`

**Comment by scott-pollom** [comment]
This should be fixed in AGP 7.0.0-beta05
